### PR TITLE
HowTo Autogenerate WebApi yaml

### PR DESCRIPTION
**Pull request by juzuluag**
Issue Number: 426

## Summary
Instructions on how to generate (manually) WebApi yaml.

## Changes

- Update WebApi README.md

## Checklist

- [ ] Local Tests Passing?
- [ ] CICD and Pipeline Tests Passing?
- [ ] Added any new Tests?
- [x] Documentation Updates Made?
- [ ] Are there any API Changes? If yes, please describe below.
- [ ] This is not a breaking change. If it is, please describe it below.

## Are there API Changes?
N/A

## Is this a breaking change?
N/A

## Anything else?
N/A

### CODE REVIEW

File: src/CarbonAware.WebApi/src/README.md [hunk 1]
```diff
@@ -165,3 +165,19 @@ Controllers are responsible for managing the "Success" responses.  If an error o
...
+## Autogenerate WebAPI
+
+Using the following steps, it is possible to get the CarbonAware WebApi OpenAPI specification
+
+1. Make sure the current directory is `CarbonAware.WebApi/src`
+
+    ```sh
+    cd src/CarbonAware.WebApi/src
+    dotnet tool restore
+    dotnet build --configuration Release --no-restore
```
Comment (gfmatthews) [inline src/CarbonAware.WebApi/src/README.md:179]: ultra technically, you need to do a 

```dotnet restore```

before you run the build and ask it not to do a restore :)

File: src/CarbonAware.WebApi/src/README.md [hunk 2]
```diff
@@ -165,3 +165,20 @@ Controllers are responsible for managing the "Success" responses.  If an error o
 The .Net framework will automatically respond to validation errors with a [ValidationProblemDetails](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.validationproblemdetails?view=aspnetcore-6.0) object.  Using the Exception Filter class enables the WebAPI to consistently respond with the `ValidationProblemDetails` error schema in all error cases and take advantage of error handling automatically provided by the framework.
 
 ![WebAPI Error Handling Flow Chart](docs/images/web-api-error-handling-flow.png)
+
+## Autogenerate WebAPI
+
+Using the following steps, it is possible to get the CarbonAware WebApi OpenAPI specification
+
+1. Make sure the current directory is `<workspace>/carbon-aware-sdk/src/`
```
Comment (bderusha) [inline src/CarbonAware.WebApi/src/README.md]: Technically there is no requirement for the root directory to share the same name as the project.  I think anyone trying to run this code will be fine with these instructions either way, but you might consider:

```suggestion
1. Make sure the current directory is `<path to project root>/src/`
```

**Review by bderusha — APPROVED** [review]
LGTM

**Review by JenMadiedo — APPROVED** [review]
🚀 -it

**Review by gfmatthews — APPROVED** [review]

**Pull request merged**